examples/se_2_3_localization.cpp

```diff
@@ -157,8 +214,9 @@ int main()
 
     // IMU measurements in IMU frame
     Vector3d alpha, omega, alpha_prev, omega_prev;
-    alpha << 10.0, 0.0, 9.80665;
-    omega << 1, 0.1, 0;
+    alpha << 10.0, 0.0, 9.80665; // constant acceleration along x-direction
```

Comment:
We should discuss a way to incorporate `g` in a less hardcoded way. We-ll leave this for later though